lib/rules/at-rule-name-case/README.md

```diff
@@ -1,5 +1,7 @@
 # at-rule-name-case
 
+> **Warning** This rule is deprecated and will be removed in the future.
+
```

Comment:
[ask] I believe the warning message is helpful, but what do you think?

---

[note] GitHub renders the `> **Warning**` text as follows:

<img width="160" alt="image" src="https://user-images.githubusercontent.com/473530/205468329-eb4c056a-5c5b-4192-a613-4443bec61585.png">

On the other hand, Docusaurus has a different rendering syntax called [Admonitions](https://docusaurus.io/docs/markdown-features/admonitions). E.g.

```markdown
:::caution

This rule is deprecated and will be removed in the future.

:::
```

If using Admonitions, we need to rewrite the warning syntax on the stylelint.io repo.

Reply:
> I believe the warning message is helpful, but what do you think?

LGTM

> we need to rewrite the warning syntax on the stylelint.io repo.

Yes, but only if it's quick to do as:

- I think it'll look good enough on the website
- we'll be removing the rules in the next major release

Reply:
Agree. It looks like we don't need to add an extra process.